connexion/middleware/security.py

```diff
@@ -31,7 +32,7 @@ def __init__(
     @classmethod
     def from_operation(
         cls,
-        operation: AbstractOperation,
+        operation: t.Union[AbstractOperation, Specification],
```

Comment:
I don't immediately understand where this is coming from: why can this now be a `Specification` object as well? The variable name is pretty confusing now :)